submissions/api.py

```diff
@@ -236,6 +236,11 @@ def get_submission(submission_uuid, read_replica=False):
         return cached_submission_data
 
     try:
+        # In previous versions of jsonfield, an error was raised whenever invalid json value was spotted,
+        # in version 3.1.0, this is changed and now a RuntimeWarning is emitted, so catching that warning
+        # to keep the endpoint response format consistent to previous version
+        warnings.filterwarnings('error')
+
```

Comment:
Changelog: https://github.com/rpkilby/jsonfield/blob/master/CHANGES.rst#v310-02222020

Commit that is changing error to warning: https://github.com/rpkilby/jsonfield/commit/c2f8780266ee32c25e2a3c3d727fba0e19d1bfb8